cluster-operator/pom.xml

```diff
@@ -270,6 +298,12 @@
                     <include>kafka-versions.yaml</include>
                 </includes>
             </resource>
+            <resource>
+                <directory>src/test/resources/io/strimzi/operator/cluster/operator/assembly/CruiseControlJSON</directory>
```

Comment:
This is a folder full of example JSON files that are returned by the mock Cruise Control REST API. It is in the pom so that the MockServer instance in the test package can easily load them.

Reply:
Can we use `<scope>testing</scope>` with resource?

Reply:
So apprently things in the `src/test/resources` are ignored by the assembly plugin unless explictly included. If I add the fully qualilifed path to the resources in the MockServer they can be loaded without the need to have a specific resource plugin entry.